src/lib/validation/utils/assignHandlersForInputs.js

```diff
@@ -21,7 +21,8 @@ function assignHandlersForSingleInput (input) {
       allConditions.forEach(condition => {
         const comparisonType = condition.indexOf('!==') > -1 ? 'isNotEqual' : 'isEqual'
         const matchers = condition.split(comparisonType === 'isNotEqual' ? '!==' : '===')
-        const form = input.closest('[data-dp-validate]')
+        const validationContainer = input.closest('[data-dp-validate]')
+        const form = validationContainer.tagName === 'FIELDSET' && validationContainer.form ? validationContainer.form : validationContainer
```

Comment:
I feel this logic needs a little commenting.

Also, i think we should move the [dp-validate documentation from demosplan-core](https://github.com/demos-europe/demosplan-documentation/blob/e11a15498b1ca8d97f519f2cc750a9914787a0a8/development/application-architecture/validation/frontend/README.md) into demosplan-ui and check if it is outdated (certainly not in this PR).

Reply:
add a comment here https://github.com/demos-europe/demosplan-ui/pull/850/commits/ae879da1bed75f3ad12d621fa9801060d9eadc93

Reply:
We currently have a Storybook in demosplan-ui. What do you mean by _move it into demosplan-ui_? create a docu for demosplan-ui?

Reply:
Correctly. At the moment we document only components and tokens via Storybook. Neither "directives", "lib" (which is highly unsorted and should be sorted upfront), "mixins" or "utils" are documented at all. At a second glance i think we should at first decide how to structure util/lib code within demosplan-ui, and only after that document things we decided to be "multi purpose enough" to stay in demosplan-ui.